doc/content/source/problems/OpenMCCellAverageProblem.md

```diff
@@ -378,6 +384,15 @@ already are exactly the same volume, then no special thought is needed for the t
 normalization, and the value will be exactly consistent with the interpretation
 used in OpenMC.
 
+## Tally and Filter Editors
+
+Cardinal provides UserObjects for editing tallies and filters.
+
+  - [OpenMCTallyEditor](https://cardinal.cels.anl.gov/source/userobjects/OpenMCTallyEditor.html)
+  - [OpenMCDomainFilterEditor](https://cardinal.cels.anl.gov/source/userobjects/OpenMCTallyEditor.html)
+
+These objects provide online control of tally and filter parameters, respectively. These objects can be used to interact with tallies and/or filters that are mapped in Cardinal as part of the `OpenMCCellAverageProblem` or are present in a `tallies.xml` file for the problem being run (unmapped tallies). These objects can be used to create tallies as well -- in this case the new objects will **not be mapped** to the mesh mirror.
```

Comment:
I would recommend revising this section of the documentation to make it clear to the user that mapped tallies cannot be modified by either `OpenMCTallyEditor` or `OpenMCDomainFilterEditor`.